src/models/serialportctrl.ts

```diff
@@ -20,70 +18,36 @@ export class SerialPortCtrl {
     return os.platform();
   }
 
-  static getComList(): Promise<SerialPortInfo[]> {
+  static async getComList() {
     return new Promise(
       (
-        resolve: (value: SerialPortInfo[]) => void,
+        resolve: (value: Object) => void,

```

Comment:
Use specific type here instead of `Object`

Reply:
Fixed.